website/docs/d/vapp_vm.html.markdown

```diff
@@ -71,6 +71,16 @@ The following arguments are supported:
 * `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
 * `vapp_name` - (Required) The vApp this VM belongs to.
 * `name` - (Required) A name for the VM, unique within the vApp 
+* `network_dhcp_wait_seconds` - (Optional; *v2.7+*) Allows to wait for up to a defined amount of
+  seconds before IP address is reported for NICs with `ip_allocation_mode=DHCP` setting. It
```

Comment:
> Allows to wait for up to a defined amount of seconds before IP address is reported for NICs with `ip_allocation_mode=DHCP` setting

One question that is left after reading this statement is whether `network_dhcp_wait_seconds` is used with other `ip_allocation_mode` modes at all. If not, we should make it clear, for example:

> This field is ignored with non-DHCP `ip_allocation_mode`.

Reply:
Just noticed that you have it formulated even more nicely here, so could just copy/paste your variant:
https://github.com/terraform-providers/terraform-provider-vcd/pull/436/files#diff-73ac9414e0dd641233165392db1d4329R79-R80

Reply:
I have rephrased website doc - do have a check if it is better.

Also - when `network_dhcp_wait_seconds` is specified and none of the adapters use DHCP - user would see something like below during READ:

```
...
vcd_vapp_vm.TestAccVcdVAppVmDhcpWaitVM: Creating...
vcd_vapp_vm.TestAccVcdVAppVmDhcpWaitVM: Still creating... [10s elapsed]
vcd_vapp_vm.TestAccVcdVAppVmDhcpWaitVM: Still creating... [20s elapsed]
vcd_vapp_vm.TestAccVcdVAppVmDhcpWaitVM: Still creating... [30s elapsed]
vcd_vapp_vm.TestAccVcdVAppVmDhcpWaitVM: Still creating... [40s elapsed]
vcd_vapp_vm.TestAccVcdVAppVmDhcpWaitVM: Still creating... [50s elapsed]
vcd_vapp_vm.TestAccVcdVAppVmDhcpWaitVM: Still creating... [1m0s elapsed]
INFO: Using 'network_dhcp_wait_seconds' only makes sense if at least one NIC is using 'ip_allocation_mode=DHCP'
...
```